### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ashishrawat93**
Final pull request for PC on QuickStarts for 10.1.1

Use authorized under Apache License 2.0

### CODE REVIEW

File: templates/informatica-powercenter-master.template [hunk 1]
```diff
@@ -598,6 +598,9 @@
                     "InformaticaDomainPassword": {
                         "Ref": "InformaticaDomainPassword"
                     },
+                    "InformaticaDomainPasswordConfirm": {
```
Comment (santiagocardenas) [inline templates/informatica-powercenter-master.template]: What are these for? Are they used?

File: templates/informatica-powercenter-linux.template [hunk 2]
```diff
@@ -3782,7 +3748,7 @@
                         {
                             "Ref": "AWS::Region"
                         },
-                        "INFAPCRHELHVM"
+                        "INFORMATICAPCRHELHVM"
```
Comment (santiagocardenas) [inline templates/informatica-powercenter-linux.template]: Please keep all of these as INFAPCRHELHVM. Thanks.

File: templates/informatica-powercenter-windows.template [hunk 3]
```diff
@@ -2989,7 +2941,7 @@
                         {
                             "Ref": "AWS::Region"
                         },
-                        "INFAPCWINHVM"
```
Comment (santiagocardenas) [inline templates/informatica-powercenter-windows.template]: Please keep all of these as INFAPCWINHVM. Thanks.

File: templates/informatica-powercenter-linux.template [hunk 4]
```diff
@@ -473,10 +470,6 @@
             "us-west-2": {
                 "Partition": "aws",
                 "QuickStartS3URL": "https://s3.amazonaws.com"
-            },
-            "ca-central-1": {
```
Comment (santiagocardenas) [inline templates/informatica-powercenter-linux.template]: Is there no support for ca-central-1 region? If there is, then we must keep this.

**Comment by ashishrawat93** [comment]
Hi,
This is just to check if the user inputs correct password two times, there is a Rule which compares the values in the two input fields.
ThanksAshish 

    On Wednesday, 22 March 2017 11:20 PM, Santiago Cardenas <notifications@github.com> wrote:
 

 @santiagocardenas commented on this pull request.In templates/informatica-powercenter-master.template:> @@ -598,6 +598,9 @@
                     "InformaticaDomainPassword": {
                         "Ref": "InformaticaDomainPassword"
                     },
+                    "InformaticaDomainPasswordConfirm": {
What are these for? Are they used?—

**Comment by ashishrawat93** [comment]
Made all changes as suggested.

**Pull request closed without merging**